packages/contracts/deploy/030_USDT_ApeLendingStrategy.deploy.ts

```diff
@@ -30,6 +30,7 @@ const func = deployOrUpgrade({
     chainlink__USDT_USD_feed, // asset token price feed
     6 * HOUR, // min report interval in seconds
     true, // Job is prepaid
+    1_500, // 15% threshold to shut down strategy
```

Comment:
I think it must be part of new health check contracts. There need only pass address of this contract

Reply:
Also, please use 5% value for threshold for now

Reply:
changed to 500 for deployment